tests/ZoneTest.php

```diff
@@ -112,4 +112,11 @@ public function testFindResourceRecords() : void
         $this->assertSame(1, $zone->find('record01.test.nl.')->count());
         $this->assertSame(0, $zone->find('record01.test.nl.', 'MX')->count());
     }
+
+    public function testGetCanonicalName(): void
+    {
+        $connector = Mockery::mock(Connector::class);
+        $zone = new Zone($connector, 'test.nl');
+        $this->assertSame('test.nl', $zone->getCanonicalName());
```

Comment:
A canonical name should end with a `.` (see my previous comment).

```suggestion
        $this->assertSame('test.nl.', $zone->getCanonicalName());
```